packages/objects/src/primitives/FarcasterAddSignature.ts

```diff
@@ -0,0 +1,7 @@
+import { Brand, make } from "ts-brand";
+
+// Farcaster's Add signature
+// https://docs.farcaster.xyz/reference/contracts/reference/key-gateway#add-signature
+
+export type FarcasterAddSignature = Brand<string, "FarcasterAddSignature">;
```

Comment:
Can we make this name even longer? FarcasterAddKeySignature? Let's get as specific as possible with these types, since they are so confusing.